tests/internal/import_hooks/test_integration.py

```diff
@@ -0,0 +1,49 @@
+import contextlib
+import mock
+import sys
+
+import pytest
+
+from ddtrace.internal import import_hooks
+
+
+@pytest.fixture
+def hooks():
+    import_hooks.hooks.reset()
+    try:
+        yield import_hooks.hooks
+    finally:
+        import_hooks.hooks.reset()
+
+
+@contextlib.contextmanager
+def remove_module():
+    was_loaded = 'urllib' in sys.modules
```

Comment:
maybe pass the module name as argument? just in case you want to test something else than `urllib` :)